site/src/components/hero.js

```diff
@@ -9,15 +9,15 @@ const Hero = () => (
     <div className="grid-container">
       <div className="usa-hero__callout">
         <h2 className="usa-hero__heading">
-          <span className="usa-hero__heading--alt">Hero callout:</span>Bring
-          attention to a project priority
+          <span className="usa-hero__heading--alt">Find Testing:</span>
+          Nearby test centers for COVID-19
         </h2>
         <p>
-          Support the callout with some short explanatory text. You don’t need
-          more than a couple of sentences.
+          Think you may have the virus or are concerned about recent exposure?
+          Fill out a couple questions to find the best course of action.
         </p>
-        <a className="usa-button" href="/">
-          Call to action
+        <a className="usa-button" href="/should-i-test">
+          Find your Course of Action
```

Comment:
Any thoughts on the wording of the button? @allisonheath  @chris-s-friedman 
This button should lead to a page that allows the user to either go throw a diagnostic flow (do you *need* testing) or simply enter a zip code for immediate test center results.
I don't want a button that reads simply as `find a test center` because that's not true, but I also don't want people to think they need to fill out a bunch of forms as may be implied by `take a quiz`.

Reply:
Maybe something more generic about testing?  `Getting Tested`?

My other thought is that this is the primary button on the front page and the one we want people to click on the first time they go to the site. So I think "Should I get tested?` may be appropriate

Reply:
Yeah, I agree that the `Should I get tested?` is the most correct, but I also would believe that people come to the website under the assumption that they *need* testing. Basically, I want to trick people into thinking that they will find testing immediately, but then convince them they may not need it if they don't hit the criteria.

Reply:
so then maybe 
```suggestion
          Getting Tested
```

Reply:
Maybe it should be
```suggestion
          Get Covid Tested
```

to match the site title?

Reply:
The matching the title is a good idea